tests/stubtest_third_party.py

```diff
@@ -167,6 +181,7 @@ def main() -> NoReturn:
     parser.add_argument("-v", "--verbose", action="store_true", help="verbose output")
     parser.add_argument("--num-shards", type=int, default=1)
     parser.add_argument("--shard-index", type=int, default=0)
+    parser.add_argument("--ci-run", action="store_true")
```

Comment:
It would be nice to add a short description of what the flag does here, using the `help=` keyword argument

Reply:
I agree on the flag name. It's a principle I need to remind myself more often:
"Don't make code that's environment-specific. Instead give options and let the environment decide".
So instead of having a "ci" flag, I should have a flag that describes this option.

I went with "specified" instead of "supported". For the same reason as the multi-platform PR by sobolevn: It's not necessarily unsupported. It's just not tested (it could simply be redundant).

Reply:
"Specified" is a great choice!